packages/components/Pagination.tsx

```diff
@@ -83,20 +96,68 @@ export const Pagination = ({
         </TouchableOpacity>
       </View>
 
-      <View
-        style={[
-          styles.section,
-          { width: edgeSectionsWidth, justifyContent: "flex-end" },
-        ]}
-      >
-        <BrandText style={styles.grayText}>Items per page:</BrandText>
-        <TextInputCustom<{ items: number }>
-          width={80}
-          name="items"
-          label=""
-          variant="labelOutside"
-          value={itemsPerPage.toString()}
-        />
+      <View style={styles.section}>
```

Comment:
```suggestion
      <View style={[
          styles.section,
          { width: edgeSectionsWidth, justifyContent: "flex-end" },
        ]}>
```

It will allow to perfect center the navigation buttons